### ISSUE TITLE
issue when repo_name does not follow naming convention

### ISSUE THREAD

**Issue by cdvv7788** [pydanny/cookiecutter-django#311]
I guess this was added to avoid name clashes, but there can be issues with some imports:

```Python
from my-report-name-with-invalid-name.models import something
```
It could be useful to force the repo_name to follow python naming conventions or mention that in the installation steps.

**Comment by pydanny** [comment]
Would you like to submit a pull request for a documentation change?

**Comment by cdvv7788** [comment]
Yeah, i can submit a little PR mentioning this.

**Comment by jayfk** [comment]
This was fixed with #375, thanks @cdvv7788!
